packages/server/modules/shared/services/paginatedItems.ts

```diff
@@ -23,6 +23,13 @@ export const getPaginatedItemsFactory =
     getTotalCount: (args: Omit<TArgs, 'cursor' | 'limit'>) => Promise<number>
   }) =>
   async (args: TArgs): Promise<Collection<T>> => {
+    if (args.limit === 0) {
+      return {
+        cursor: null,
+        totalCount: 0,
```

Comment:
that's not what I meant, the totalCount should still be calculated. the point of passing in limit===0 is to optimize execution - skip items, but still get the total count

Reply:
you're right, my bad